src/app/modules/shared-components/components/add-content/add-content.component.ts

```diff
@@ -9,6 +11,7 @@ import { Subscription } from 'rxjs';
 })
 export class AddContentComponent implements OnInit, OnDestroy {
   @Input() allowSkills = false;
+  @Output() childrenAdded = new EventEmitter<NewItem>();
```

Comment:
This is not necessarily children in other scenarios. So, "contentAdded" seems more appropriate.

Reply:
Done